docs/contents/components/data-display/stat/accessibility.ja.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Stat
+description: "`Stat`は、数値やデータをボックス内に表示するために使用されます。"
+package_name: "@yamada-ui/stat"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+## ARIAロールと属性
+
+| コンポーネント | ロールと属性 | 使い方                                                                                                                 |
+| -------------- | ------------ | ---------------------------------------------------------------------------------------------------------------------- |
+| `Stat`         | `icon`       | `"increase"`または`"decrease"`が設定されている場合、`StatIcon`に`type`として渡されます。                               |
+| `StatIcon`     | `type`       | `"increase"`または`"decrease"`が設定できます。                                                                         |
```

Comment:
There is no `type` role or attributes.This is unique to Yamada UI.Please remove this line.